linera-execution/src/runtime.rs

```diff
@@ -33,12 +33,20 @@ use std::{
 /// Runtime data tracked during the execution of a transaction.
 #[derive(Debug, Clone)]
 pub(crate) struct ExecutionRuntime<'a, C, const WRITABLE: bool> {
+    /// The amount of fuel available for executing the application.
```

Comment:
Can we group the resource fields? Also I would generally add new fields at the end if there is no obvious choice

Reply:
Bu grouping, do you mean replace the `Arc<Atomic...>` by `Arc<Mutes<RuntimeCounts>>`?